### DISCUSSION TITLE
NER识别支持跨文档抽取与预训练吗？

### DISCUSSION THREAD

**Discussion by springwings** [PaddlePaddle/PaddleNLP#2238] [General]
NER识别支持跨文档抽取与预训练吗？
还有长文本标注的时候也是如下方式吗？
{
    "id": 36,
    "text": "2月8日上午北京冬奥会自由式滑雪女子大跳台决赛中中国选手谷爱凌以188.25分获得金牌",
    "relations": [],
    "entities": [
        {
            "id": 47,
            "start_offset": 0,
            "end_offset": 6,
            "label": "时间"
        },
        {
            "id": 48,
            "start_offset": 6,
            "end_offset": 23,
            "label": "赛事名称"
        },
        {
            "id": 49,
            "start_offset": 28,
            "end_offset": 31,
            "label": "选手"
        },
        {
            "id": 50,
            "start_offset": 32,
            "end_offset": 39,
            "label": "得分"
        }
    ]
}

**Comment by ZeyuChen** [comment]
@wawltor
